commands/commands.go

```diff
@@ -100,7 +100,8 @@ func bindFlags() error {
 	pf.Bool("disablerpc", uc.DisableRPC, "disable RPC server")
 	pf.Bool("disabletls", uc.DisableTLS, "disable TLS for the RPC server")
 
-	pf.Int64("confirmations", uc.Confirmations, "Number of block confirmations ")
+	pf.Uint32("confirmations", uc.Confirmations, "Number of transaction confirmations ")
+	pf.Uint32("coinbaseMaturity", uc.CoinbaseMaturity, "Number of CoinBase confirmations ")
```

Comment:
This is a one-time network configuration parameter. Different networks have different values. So it can be read directly from `ActiveNet.CoinbaseMaturity`(https://github.com/Qitmeer/qitmeer-wallet/blob/0.10-dev/config/config.go#L96)